### ISSUE TITLE
Missing icons in inventory and on logic blocks

### ISSUE THREAD

**Issue by MormegiI** [tsunamayo/Starship-EVO#2756]
### Short Description of the issue:
My Logic blocks are all missing the icons they should show and are not differentiable.
Also the Window-Bricks in inventory have no icons, as do the markings. See attached pictures:

![grafik](https://user-images.githubusercontent.com/51251532/89437609-164d4000-d748-11ea-8cec-c849921b4076.png)
![grafik](https://user-images.githubusercontent.com/51251532/89437635-1cdbb780-d748-11ea-9dc1-7c48be6e4506.png)
![grafik](https://user-images.githubusercontent.com/51251532/89437640-206f3e80-d748-11ea-99b9-4f4f6aa9097e.png)

### Steps to reproduce the issue:
Be me, start the game and check inventory/logic on ships.
No idea, honestly.

### Environment
- Starship EVO version: current and previous on experimental branch. Not checked stable yet.
- Graphics Card: Geforce 920M (yes, bit older Laptop)

### Additional information:

See screenshots above.

Output Log: 
[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/5029721/output_log.txt)

**Comment by tsunamayo** [comment]
@MormegiI When does this appeared? Did you had this 2-3 weeks ago or maybe you only checked the game recently? 
I dont have any issue on my GPU... Could you tell me if you see some shadow? It seems that this is the same issue as #2727.
 Thanks

**Comment by tsunamayo** [comment]
Also it seems that your drivers are very old (by the look of it April 2018), could you update them and tell me how it worked?

**Comment by MormegiI** [comment]
Yeah i came back to the game a few days ago. Noticed the bug only on my Laptop, my PC works fine.
I can't access the laptop for the next few days though. Will update the drivers on the Laptop and report next week. I don't think i saw any shadows when i think about it, though.

**Comment by tsunamayo** [comment]
@MormegiI what is your OS version? Thanks

**Comment by MormegiI** [comment]
Windows 10, 1910 i think.

**Comment by tsunamayo** [comment]
@MormegiI so could you please tell me how it goes in the new Experimental beta 20w34a? Thanks

**Comment by MormegiI** [comment]
At least on my Laptop the Bug is persisting. No icons on logic and invisible gratings.
If you need any data/files from me, just tell me and I'll send it to you.

**Comment by T3hWombat** [comment]
I want to add that I bought the game today and I am having this exact problem as well. I tried multiple resolutions, thinking it could be a texture scaling problem. I have a 4k monitor, but usually play stuff in fullscreen 1080p to give my older GTX660Ti a fighting chance. No combination of graphics settings or screen resolutions seemed to help. It seems to affect any glass blocks as well - they appear totally invisible, but if bloom is enabled I can see a reflection off the invisible glass.

I updated my graphics drivers, verified I have the latest Windows 10 updates, etc.
